zulipterminal/model.py

```diff
@@ -1374,7 +1374,7 @@ def _handle_typing_event(self, event: Event) -> None:
 
         narrow = self.narrow
         controller = self.controller
-        active_conversation_info = controller.active_conversation_info
+        active_conversation_info = self.controller.active_conversation_info
```

Comment:
It's unclear to me why this change is necessary. The controller local variable is defined above this.

Reply:
My bad . I thought one way to clear the attribute error(in tests) could be to directly assign it (using self.controller.active_convesation_info rather than passing it from a variable). But that didn't work. So, this change is not necessary and I will fix it.